scipy/stats/_continuous_distns.py

```diff
@@ -845,10 +845,19 @@ class betaprime_gen(rv_continuous):
     for :math:`x >= 0`, :math:`a > 0`, :math:`b > 0`, where
     :math:`\beta(a, b)` is the beta function (see `scipy.special.beta`).
 
+    The distribution is related to the `beta` distribution as follows:
+    If :math:`X` follows a beta distribution with parameters :math:`a, b`,
+    then :math:`Y = X/(1-X)` has a beta prime distribution with
+    parameters :math:`a, b` ([1]_).
+
     `betaprime` takes ``a`` and ``b`` as shape parameters.
 
     %(after_notes)s
 
+    References
+    ----------
+    .. [1] https://en.wikipedia.org/wiki/Beta_prime_distribution
```

Comment:
We're not super formal, but IIRC i've seen this encouraged somewhere:
``` suggestion
    .. [1] Beta prime distribution, Wikipedia,
           https://en.wikipedia.org/wiki/Beta_prime_distribution
```

Reply:
done